clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -342,13 +342,13 @@ class CIRGenBuilderTy : public CIRBaseBuilderTy {
   }
   bool isInt(mlir::Type i) { return i.isa<mlir::cir::IntType>(); }
 
-  mlir::FloatType getLongDouble80BitsTy() const {
+  mlir::cir::FloatType getLongDouble80BitsTy() const {
     return typeCache.LongDouble80BitsTy;
```

Comment:
You don't need to worry about this one just yet, you can tackle this in upcoming PRs.